src/library/trackset/baseplaylistfeature.cpp

```diff
@@ -151,20 +157,34 @@ int BasePlaylistFeature::playlistIdFromIndex(const QModelIndex& index) {
     }
 }
 
+void BasePlaylistFeature::selectPlaylistInSidebar(int playlistId) {
+    if (playlistId == -1) {
+        return;
+    }
+    QModelIndex index = indexFromPlaylistId(playlistId);
+    if (index.isValid() && m_pSidebarWidget) {
+        m_pSidebarWidget->selectChildIndex(index);
+    }
+}
+
 void BasePlaylistFeature::activateChild(const QModelIndex& index) {
     //qDebug() << "BasePlaylistFeature::activateChild()" << index;
     int playlistId = playlistIdFromIndex(index);
     if (playlistId != -1) {
         m_pPlaylistTableModel->setTableModel(playlistId);
         emit showTrackModel(m_pPlaylistTableModel);
         emit enableCoverArtDisplay(true);
+        if (m_pSidebarWidget) {
```

Comment:
Same here, check m_pSidebarWidget and otherwise early return when entering this function.

Reply:
Thats not possible here, because this function is called in init when m_pSidebarWidget is not populated yet.